overrides.rb

```diff
@@ -47,5 +47,9 @@
         if !cxx_flags.empty?
             pkg.define "CMAKE_CXX_FLAGS", cxx_flags
         end
+
+        if pkg.depends_on("external/sisl")
```

Comment:
This would be `pkg.depends_on?`. You are adding `external/sisl` as dependency to all packages.

Reply:
You are right, I didn't realize that.

OK, the pkg seems to have no dependencies known to autobuild, although they are in the manifest. Maybe something has to be loaded?

```bash
[4] pry(main)> pkg.dependencies
=> []
```

Reply:
Actually, I believe the issue is that *when using the debian packages*, there should be an override file that properly sets all the packages `prefix` to the Debian package prefix. SISL_PREFIX is already set by rock.core. The issue is that the package's `prefix` is wrong.

Just set the prefix for the packages that are maintained by the Debian packages to `/opt/rock/... ` and things should be working. For any cross-package references.